website/src/pages/case-studies/nutanix-fixed-log4j-with-sourcegraph.tsx

```diff
@@ -309,7 +309,7 @@ const NutanixCaseStudy: FunctionComponent<PageProps> = props => (
 
             <AuthorBio
                 customer="Nutanix"
-                image="/case-studies/jon-kohler.png"
+                image="/case-studies/jon-kohler-circle.png"
```

Comment:
Instead of introducing a dupe asset just for a radius crop, I think we should use the square image since the img element for the author already has a 50% border radius for cropping the overflow. We'll just have to update the width/height ratio for the img element itself.

For future, this is one of those things where we can add validation for image dimensions for an author image content field in our Contentful models.

cc: @zlonko

Reply:
Right, good call! I just bothered you about dupe assets a few weeks ago haha, I forgot already 😅